### PR TITLE
Add tpr option to gmx tools; fix output filter

### PR DESCRIPTION
**Pull request by simonbray**
@gmauro and I figured out what was wrong with the pulsar jobs - the output filtering was not working in the wrapper, and pulsar couldn't deal with the missing outputs.

Also, I added an option to return a tpr file; this is required for some post-processing of the trajectory files.

### CODE REVIEW

File: tools/gromacs/sim.xml [hunk 1]
```diff
@@ -257,10 +265,12 @@
             <!-- <param name="ndx_bool" value="false" /> -->
             <param name="traj" value="trr"/>
             <param name="str" value="gro"/>
+            <param name="tpr_out" value="true"/>
```
Comment (bgruening) [inline tools/gromacs/sim.xml]: can you add num_outputs to the tests so that you test the filtering?

Comment (simonbray) [reply tools/gromacs/sim.xml]: Good idea, thanks!

**Comment by simonbray** [comment]
Thanks, can we update this straight away on usegalaxy.eu? Then we can try sending my workflow to GB again.

**Comment by bgruening** [comment]
@simonbray can you check if its installed?

**Comment by simonbray** [comment]
Yes, looks good. Testing it now. Thanks!

2019-12-18 22:15 GMT+01:00, Björn Grüning <notifications@github.com>:

**Pull request merged**